android/src/main/kotlin/im/nfc/flutter_nfc_kit/ByteUtils.kt

```diff
@@ -35,4 +39,38 @@ object ByteUtils {
         val secondIndex = octet and 0x0F
         return "${HEX_CHARS_ARRAY[firstIndex]}${HEX_CHARS_ARRAY[secondIndex]}"
     }
+
+    // MifareClassic
```

Comment:
Mifare related methods should not be placed in this file. It would be better to create a new file.

Reply:
I suggest extracting to a file, maybe named 'MifareUtils' or something like that

Reply:
okay

Reply:
Maybe these methods do not need to be extracted. Seems for `mifareClassis`, it equals to read four consecutive blocks. For `mifareUltralight`, it just read one page.